<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FicheAccesDistance</class>
 <widget class="QWidget" name="FicheAccesDistance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fiche d'accès à distance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QGroupBox" name="filialeGroup">
       <property name="title">
        <string>Filiale</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="accessDistance_filiale_lineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="dateGroup">
       <property name="title">
        <string>Date</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QDateEdit" name="accessDistance_date_dateEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Fiche d'accès à distance</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom et Prénom :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="accessDistance_nomPrenom_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Service :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="accessDistance_service_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="accessGroup">
     <property name="title">
      <string>Période</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QRadioButton" name="accessDistance_periodeTemp_radio">
        <property name="text">
         <string>Accès temporaire </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Période</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Type d’accès</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="accessDistance_periodePerm_radio">
        <property name="text">
         <string>Accès permanent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="accessDistance_typeAccesTotal_radio">
        <property name="text">
         <string>Accès totale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="accessDistance_typeAccesPersonalise_radio">
        <property name="text">
         <string>Accès Personnalisé</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="needsGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_app1">
          <property name="text">
           <string>Besoin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="accessDistance_besoin_textEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_app2">
          <property name="text">
           <string>Applications et systèmes à utiliser</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="accessDistance_appSysteme_textEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="legalText">
     <property name="text">
      <string>L'accès distant est destiné uniquement à des utilisations professionnelles en relation directe avec la nature des fonctions du demandeur. Les mots de passe ne doivent pas être communiqués à d'autres personnes. En cas de perte du TOKEN l'utilisateur devra prendre en charge son renouvellement.

Je m'engage à respecter les procédures et règles de sécurité informatique en vigueur.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tokenGroup">
     <property name="title">
      <string>Cie de sécurité</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Token</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDateEdit" name="accessDistance_cleSecurite_token_dateDebut_dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>S/N :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Période souhaitée</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>S/N :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="accessDistance_cleSecurite_deviceMobile_lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Device mobile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="accessDistance_cleSecurite_token_lineEdit"/>
      </item>
      <item row="1" column="4">
       <widget class="QDateEdit" name="accessDistance_cleSecurite_token_dateFin_dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Du :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Au :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="accessDistance_submit_pushButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Soumettre la demande</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
